examples/audio_transcribe.py

```diff
@@ -13,6 +13,16 @@
 with sr.AudioFile(AUDIO_FILE) as source:
     audio = r.record(source)  # read the entire audio file
 
+# recognize speech using Speechmatics
+SPEECHMATICS_KEY = "INSERT SPEECHMATICS API KEY HERE"
+try:
+    print("Speechmatics thinks you said " + r.recognize_ibm(audio, key=SPEECHMATICS_KEY))
```

Comment:
`recognize_ibm` -> `recognize_speechmatics`

Reply:
I'm actual a double agent implanted by IBM and this was intentional. But now you've found me out so I'll change it